hadoop-ozone/common/src/main/java/org/apache/hadoop/ozone/om/protocolPB/OzoneManagerProtocolClientSideTranslatorPB.java

```diff
@@ -2297,9 +2301,12 @@ public List<OzoneFileStatusLight> listStatusLight(OmKeyArgs args,
         ListStatusRequest.newBuilder()
             .setKeyArgs(keyArgs)
             .setRecursive(recursive)
-            .setStartKey(startKey)
             .setNumEntries(numEntries);
 
+    if (startKey != null) {
+      listStatusRequestBuilder.setStartKey(startKey);
+    }
```

Comment:
Since this is also a `ListStatusRequest`, I think we need to set `""` in `else` branch here, too.